open-policy-agent/infrastructure-general/namespace-regulation-for-pods/README.md

```diff
@@ -0,0 +1,33 @@
+# Namespace Regulation for Pods
+
+The policy disallows pods from running on specific namespaces. The disallowed namespaces must be provided as a parameter.
+
+Running workload on undesired namespace can be a potential security risk as they might run without being monitored and regulated.
+Placing objects in the default namespace, for example, makes application of RBAC and other controls more difficult.

```

Comment:
Not sure I understand what is meant by `makes application of RBAC and other controls more difficult`.